src/main/java/org/jabref/gui/bibtexkeypattern/BibtexKeyPatternPanel.java

```diff
@@ -78,21 +73,15 @@ private void buildGUI() {
         }
 
         addExtraText(mode);
-        int y = 0;
-        for (EntryType type : EntryTypes.getAllValues(mode)) {
-            textFields.put(type.getName().toLowerCase(Locale.ROOT), textFieldArray[y]);
-            y++;
-        }
 
         Button help1 = new Button("?");
         help1.setOnAction(e->new HelpAction(Localization.lang("Help on key patterns"), HelpFile.BIBTEX_KEY_PATTERN).getHelpButton().doClick());
```

Comment:
You need to move this help and the reset all button beneath the loop, with rowindex+3 otherwise it will be somewhere in the middle because it still uses hardcoded value. That's what I discovered while fixing it in the other PR.